frontend/src/services/userProfiles.js

```diff
@@ -53,18 +54,49 @@ UserProfilesService.getRecruiters = idToken => {
     return _axios.get('/user-profiles/recruiters', config(idToken))
 }
 
+
 UserProfilesService.getUserProfilesByTeamId = (teamId, idToken) => {
     return _axios.get(`/user-profiles/team/${teamId}`, config(idToken))
 }
 
+
+
 UserProfilesService.updateRecruiter = (
+    idToken,
+    recruiterId,
+    event,
+    organisation,
+) => {
+    const data = { recruiterId, event, organisation }
+    return _axios.patch('/user-profiles/recruiters', data, config(idToken))
+}
+
+UserProfilesService.deleteRecruiter = (
+    idToken,
+    recruiterId,
+    event,
+
+) => {
+    const data = { recruiterId, event }
+    return _axios.patch('/user-profiles/recruiters/delete', data, config(idToken))
+}
+
+UserProfilesService.updateRecruiterAdmin = (
     idToken,
     recruiterId,
     events,
     organisation,
 ) => {
     const data = { recruiterId, events, organisation }
-    return _axios.patch('/user-profiles/recruiters', data, config(idToken))
+    return _axios.patch('/user-profiles/recruiters/admin', data, config(idToken))
+}
+
+UserProfilesService.deleteRecruitersAdmin = (
+    idToken,
+    recruiterId,
+) => {
+    const data = { recruiterId, }
+    return _axios.patch('/user-profiles/recruiters/admin/delete', data, config(idToken))
```

Comment:
- [ ] Optional: It is recommended to use the HTTP method that corresponds to the action being performed to follow the RESTful principles

Reply:
was done previously with PATCH so I'm sticking with that for time purposes. Also it is admin side witch will be deprecated in the future.